src/components/Layout/Header/Header.tsx

```diff
@@ -10,16 +10,17 @@ import {
   useColorModeValue,
```

Comment:
No need to review anything from here and down. It's the changes from this PR: https://github.com/shapeshift/web/pull/2717